### DISCUSSION TITLE
[자료구조 2일차] 스택, 큐, 덱

### DISCUSSION THREAD

**Discussion by seongeun42** [Shortudy-10th/tech-for-developer#31] [오늘의 주제]
스택, 큐, 덱 관련 질문 2인조 1개, 3인조 2개씩 총 5개

**Comment by watty-exclamationMark** [comment]
큐, 스택, 덱이 각각 어떠한 구조를 가지는지 설명해 주세요.

**Comment by sikutisa** [comment]
스택, 큐, 덱을 프로그램상에서 구현한다면, 어떻게 구현할 수 있을까요?

**Comment by sikutisa** [comment]
OS, 어플리케이션 등 다양한 프로그램에서 스택, 큐, 덱을 실제로 어떻게 활용하고 있나요? 그 예시에 대해 설명해주세요.

**Comment by LireEruel** [comment]
브라우저에서 스택과 큐는 언제 활용되고 있나요?

**Comment by Jihn0118** [comment]
스택과 큐의 특징과 이 둘은 어디에 주로 사용되나요?
